### PR TITLE
fix: 2.0 pfe-jump-links doc updates and fixes

### PR DESCRIPTION
**Pull request by heyMP**
- Update docs
- Fix that enables user to set the offset to `0`
- Fix active item indicator

### CODE REVIEW

File: elements/pfe-jump-links/pfe-jump-links-nav.scss [hunk 1]
```diff
@@ -130,7 +130,13 @@ li {
 
   // Active state bar; uses pseudo-element, not border, to prevent angled top/bottom
   &::before {
-    @include accent-bar(vertical);
+    position: absolute;
```
Comment (bennypowers) [inline elements/pfe-jump-links/pfe-jump-links-nav.scss:133]: love this change

File: elements/pfe-jump-links/pfe-jump-links-nav.scss [hunk 2]
```diff
@@ -130,7 +130,13 @@ li {
...
+    content: "";
+    top: 0px;
+    left: 0px;
+    height: calc(100% - 0px);
+    width: calc(var(--pfe-theme--surface--border-width--heavy,4px) - 0px);
+    background-color: transparent;
```
Comment (bennypowers) [inline elements/pfe-jump-links/pfe-jump-links-nav.scss:139]: is this redundant?

File: elements/pfe-jump-links/pfe-jump-links-nav.ts [hunk 3]
```diff
@@ -38,13 +38,16 @@ function isQueryable(x: Node): x is Document|ShadowRoot {
 
 /**
  * @fires pfe-jump-links-panel:active-navItem
```
Comment (bennypowers) [inline elements/pfe-jump-links/pfe-jump-links-nav.ts:40]: can we deprecate the el:name events and add our own `extends Event` events?

File: elements/pfe-jump-links/pfe-jump-links-nav.ts [hunk 4]
```diff
@@ -333,7 +336,7 @@ export class PfeJumpLinksNav extends LitElement {
   // @TODO It seems like the offset is 0 when non-horizontal jumps links are mobile
   get offsetValue(): number {
     // If the offset attribute has been set, use that (no calculations)
-    if (this.offset) {
+    if (typeof this.offset !== 'undefined' && this.offset !== null) {
```
Comment (bennypowers) [inline elements/pfe-jump-links/pfe-jump-links-nav.ts:339]: could also be `this.offset != null`

File: elements/pfe-jump-links/pfe-jump-links-nav.scss (hunk 2 above)
Comment (heyMP) [reply elements/pfe-jump-links/pfe-jump-links-nav.scss:139]: yes! good catch.

File: elements/pfe-jump-links/pfe-jump-links-nav.scss (hunk 1 above)
Comment (heyMP) [reply elements/pfe-jump-links/pfe-jump-links-nav.scss:133]: 😎

File: elements/pfe-jump-links/pfe-jump-links-nav.ts (hunk 3 above)
Comment (heyMP) [reply elements/pfe-jump-links/pfe-jump-links-nav.ts:40]: Good idea.  Just updated them to our new `ComponedEvent`

**Pull request merged**